custom/enikshay/integrations/nikshay/repeaters.py

```diff
@@ -216,7 +216,7 @@ def allowed_to_forward(self, episode_case):
 
         episode_case_properties = episode_case.dynamic_case_properties()
         return (
-            episode_case_properties.get('nikshay_registered', 'false') == 'false' and
+            episode_case_properties.get('private_nikshay_registered', 'false') == 'false' and
```

Comment:
that was a bad miss.. :(
thanks for catching.

Did this result in irrelevant triggers?